builders/build/historical-updates.md

```diff
@@ -462,6 +462,20 @@ This migration was executed at the following runtimes and blocks:
 
 For more information, you can review the [relative PR on GitHub](https://github.com/moonbeam-foundation/moonbeam/pull/2685){target=\_blank}.
 
+---
+
+### EVM Pallet {: evm-pallet }
+
+#### EVM Contract Metadata
+
+A migration was introduced to automate the manual process of setting EVM contract metadata for contracts deployed more than two years ago that hadn't been interacted with after the introduction of metadata storage item. This migration replaces the need to manually call `createContractMetadata(address)` on these contracts to make them compatible with the current runtime. 
+
+This migration was executed at the following runtimes and blocks:
+
+| Network  | Executed Runtime | Block Applied |
```

Comment:
Can you also add the details for Moonriver? This was applied to Moonriver as well :)